docs/changelog/2023-09-29.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Sep 29, 2023
+tags: []
+---
+
+## Overview
+
+### Features & Improvements
+
+1. Visitors will now be able to carry on conversations over email
+2. Widget summary page!
+3. Improve website scraping robustness
+4. Fixed random name / colour generator for new visitors occasionally selecting an invalid value
+5. Improved billing report accuracy
```

Comment:
I would move this to a bug fixes section (similar to this: https://github.com/algomo/knowledge-base/pull/18/files)

Reply:
and for the other bugs too